### PR TITLE
Added:Get/Set propElasticity

### PR DESCRIPTION
**Pull request by dvdvideo1234**
According to the Gmod/Entity Methods wiki:

http://wiki.garrysmod.com/page/Entity/GetElasticity
http://wiki.garrysmod.com/page/Entity/SetElasticity

### CODE REVIEW

File: lua/entities/gmod_wire_expression2/core/custom/prop.lua [hunk 1]
```diff
@@ -326,6 +326,16 @@ e2function number entity:propGetFriction()
 	return this:GetFriction()
 end
 
+e2function void entity:propSetElasticity(number elasticity)
```
Comment (dvdvideo1234) [inline lua/entities/gmod_wire_expression2/core/custom/prop.lua]: Just like the friction above ...

File: lua/entities/gmod_wire_expression2/core/custom/prop.lua [hunk 2]
```diff
@@ -326,6 +326,16 @@ e2function number entity:propGetFriction()
...
+	if not PropCore.ValidAction(self, this, "elasticity") then return end
+	this:SetElasticity( math.Clamp(elasticity, -1000, 1000) )
```
Comment (AbigailBuccaneer) [inline lua/entities/gmod_wire_expression2/core/custom/prop.lua]: Does a negative elasticity have any effect?

**Comment by dvdvideo1234** [comment]
I don't know, never tried it ...

**Comment by AbigailBuccaneer** [comment]
Please try giving an entity a negative elasticity, to see if it has any effect. At the very least check that it doesn't cause any errors or glitches.

**Comment by dvdvideo1234** [comment]
I will be sure to check it out

**Comment by dvdvideo1234** [comment]
Ir has slight effect on the props. Tested with the washing machine and a helicopter part. Garry uses it like so: 

https://github.com/garrynewman/garrysmod/search?utf8=%E2%9C%93&q=Entity.SetElasticity

Still It does not crash with the following values because of the clamp:

-2000,-1000, -500 -10 -1 0 1 50 500 1000 2000

**Pull request merged**